python/eos/tasks.py

```diff
@@ -425,6 +425,34 @@ def run(analysis_file:str, base_directory:str='./', dry_run:bool=False, executor
         raise e
```

Comment:
Can you please reword the commit to spell out ``nested sampling``? This makes it easier to find this commit.